assets/js/form.js

```diff
@@ -0,0 +1,47 @@
+// whole form
+// view
+var FormView = Backbone.View.extend({
+  template: AppTemplates.form,
+
+  el: '#target',
+
+  initialize: function() {
+    this.render();
+  },
+
+  events: {
+    'submit form': 'createPerson'
+
+    // 'form .submit': 'submitForm'
+  },
+
+  createPerson: function(ev) {
+    ev.preventDefault();
+
+    var firstName = this.$('#first-name').val();
+    var lastName = this.$('#last-name').val();
+    var address = this.$('#address').val();
+    var phone = this.$('#phone').val('');
```

Comment:
Note the following screenshot: 
![screen shot 2015-08-16 at 10 16 58 pm](https://cloud.githubusercontent.com/assets/2532004/9297395/9038d400-4464-11e5-9d52-4a02e18720b0.png)